packages/ui/src/utils/typography.ts

```diff
@@ -0,0 +1,126 @@
+import { css } from 'styled-components';
+
+/**
+ * This file contains styles that are used throughout the Penumbra UI library.
+ * Many of them correlate 1-to-1 to specific components (such as `h1`, `large`,
+ * etc.), while others are base styles shared by a number of components.
+ */
+
+const base = `
+  margin: 0;
+`;
+
+export const h1 = css`
+  ${base}
+
+  font-family: ${props => props.theme.font.heading};
+  font-size: ${props => props.theme.fontSize.text6xl};
+  font-weight: 500;
+  line-height: ${props => props.theme.lineHeight.text6xl};
+`;
```

Comment:
suggestion: rename the components (or Typography's type prop) to align with the design. For example, make `type: 'text-6xl'` be rendered as h1.

I imagine developers working with the designs will see the 'text-6xl' property of some text in the wireframes, they simply put 'text-6xl' in the code and expect the component to render everything correctly

Reply:
So, these are named to match the [typography _semantics_](https://www.figma.com/design/WKgXSVVK3L0EI0sr9Hwjoa/penumbra-UI-library?node-id=37-7542&m=dev), rather than the [typography _primitives_](https://www.figma.com/design/WKgXSVVK3L0EI0sr9Hwjoa/penumbra-UI-library?node-id=1-112&m=dev). The semantics are designed to group the primitives (such as the 6xl font size, 6xl line height, 500 weight, and heading font) into semantically meaningful groups. 

Figma shows users both the semantics and primitives in dev mode. In this screenshot, the semantic name is the CSS comment (`tab`), while the primitives are the individual rules:
<img width="298" alt="image" src="https://github.com/user-attachments/assets/408b429f-7436-4bf6-a7da-73e872ee8683">


Also, more than one semantic grouping share the same design

Reply:
got it, thanks for explaining